src/policy/CMSRucioPolicy/algorithms/pfn2lfn.py

```diff
@@ -0,0 +1,14 @@
+"""
+PFN to scope:name translation algorithm for CMS Rucio policy
+"""
+from collections.abc import Mapping
+
+def cms_pfn2lfn(parsed_pfn: Mapping[str, str]) -> tuple[str, str]:
+    """
+    This function converts a PFN into scope:name pair.
+    It ignores user scopes and sets all scopes to cms.
+    """
+    # Ignore user scopes for now
+    scope = "cms"
+    name = parsed_pfn['path'] + parsed_pfn['name']
```

Comment:
Hmm, I do not seem to understand this. 
The `lfn` would only be "name" and not a concat of `path` and `name`.

Reply:
Not really, this is what `parsed_pfn` looks like for an example cms PFN:
```
{'scheme': 'davs', 'port': '1094', 'hostname': 'cmsxrootd.hep.wisc.edu', 'path': '/store/backfill/1/CMSSW_12_3_0/DoubleMuon_New/FEVTDEBUGHLT/HLTDR2_2018_SC_LumiMask_Rules_June2024_Val_v1-v11/00000/', 'name': 'e6f95196-5df4-4d03-a786-2fd3fcb968b1.root', 'prefix': '/store/temp/sitetest', 'web_service_path': ''
...
```

Reply:
Ok perfect!